vm-setup/roles/v1aX_integration_test/tasks/node_reuse.yml

```diff
@@ -61,6 +118,9 @@
       definition:
         spec:
           version: "{{ UPGRADED_K8S_VERSION }}"
+          machineTemplate:
+            infrastructureRef:
+              name: "{{CLUSTER_NAME}}-new-controlplane-image"
           rolloutStrategy:
             rollingUpdate:
               maxSurge: 0
```

Comment:
Do we need to hard code maxSurge value here? We have MAX_SURGE_VALUE environment variable to set this value when creating the controlplane. https://github.com/metal3-io/metal3-dev-env/blob/3423a5039f4cb939c3a8d5cf2d32b1b194172cd0/vm-setup/roles/v1aX_integration_test/templates/cluster-template-controlplane.yaml#L9

Reply:
Will do it separate refactoring PR.